src/screens/FeedScreen/RenderList.js

```diff
@@ -226,7 +225,8 @@ const RenderListFeed = (props) => {
           withToast={true}
           isVisible={isBlurred}
           containerStyle={{
-            height: commentHeight()
+            height: isBlurred || !hasComment ? addCommentPreviewHeight() : undefined,
+            marginBottom: hasComment ? normalize(4) : 0
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The height of the container is now set to `addCommentPreviewHeight()` if `isBlurred` or `!hasComment` is true, otherwise it's undefined. Also, the marginBottom is now set to `normalize(4)` if `hasComment` is true, otherwise it's 0. Make sure these changes do not affect the layout of the container.
```diff
-            height: commentHeight()
+            height: isBlurred || !hasComment ? addCommentPreviewHeight() : undefined,
+            marginBottom: hasComment ? normalize(4) : 0
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->